### ISSUE TITLE
2.9.1 Fieldtype [title] does not exist

### ISSUE THREAD

**Issue by PeterEmil** [statamic/v2-hub#1943]
After updating from 2.8.12 to 2.9.1 using `php please:update` in the terminal & login the CP & travel to a Page / Collection Entry this is the first message I get:
```
Whoops, looks like something went wrong.

1/1
FatalException in FieldtypeFactory.php line 45:
Fieldtype [title] does not exist.
in FieldtypeFactory.php line 45
at FieldtypeFactory::create('title', array('type' => 'title', 'localizable' => true, 'name' => 'title')) in Fieldset.php line 283
at Fieldset->fieldtypes() in Processor.php line 19
...
```
Before updating all was fine. And I have titles in my `.md` files.

Steps to reproduce:
1. update in terminal
2. login to dashboard CP
3. go to a Page or Collection Entry

**Comment by PeterEmil** [comment]
thanks! I thought `php please:update` didn't need any housekeeping afterwards, but there you go

**Comment by jasonvarga** [comment]
It shouldn't! But it wasn't working correctly. It got fixed in 2.9.1
See https://github.com/statamic/v2-hub/issues/1928
